src/schemas/json/golangci-lint.json

```diff
@@ -3071,8 +3072,62 @@
               "required": ["path"]
             }
           }
+        },
+        "perfsprint": {
```

Comment:
@ldez I think I removed everything related to unreleased code (master branch), but I am on doubt about these two linters: they appear in linters list for 1.55.2 (ref https://github.com/golangci/golangci-lint/blob/v1.55.2/.golangci.reference.yml#L2317 and https://github.com/golangci/golangci-lint/blob/v1.55.2/.golangci.reference.yml#L2321) but not in linters-settings https://github.com/golangci/golangci-lint/blob/v1.55.2/.golangci.reference.yml#L110

Should they be here? Are these rules I added correct?

Reply:
the options have been added after v1.55.2, so they should be removed.

Reply:
done! :smile: